src/style.css

```diff
@@ -0,0 +1,82 @@
+:root {
+  --border-color: #e1e7ed;
+  --background-color: #fff;
+  --dark-color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", Arial, sans-serif;
+  margin: 0;
+  padding: 0 8px;
+}
+
+.search__form {
+  margin-top: 20px;
+}
+
+.search__input {
+  width: 100%;
+  font: inherit;
+  background: var(--background-color);
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+}
+
+.search__input--big {
+  text-indent: 62px;
+  font-size: 16px;
+  line-height: 100%;
+  height: 70px;
+}
```

Comment:
move this styles to `.search__input` class, there is no need for `--big` modifire